src/input.cpp

```diff
@@ -509,6 +509,10 @@ char_event_t inputter_t::readch(bool allow_commands) {
             // If we have EOF, we need to immediately quit.
             // There's no need to go through the input functions.
             return evt;
+        } else if (evt.is_idle()) {
```

Comment:
idle can theoretically also pass through the input functions. It will may cause a debug message so I catch it here.
Is this correct?

Reply:
I would propose to leave it like this. And later, if we have idle handler, add the logic here.